cordapp-cpk2/src/main/kotlin/net/corda/plugins/cpb2/CpbPlugin.kt

```diff
@@ -94,8 +88,8 @@ class CpbPlugin : Plugin<Project> {
             }
         }
 
-        val cpkTask = project.tasks.named(CPK_TASK_NAME, PackagingTask::class.java)
-        val cpkPath = cpkTask.flatMap(PackagingTask::getArchiveFile)
+        val cpkTask = project.tasks.named(JAR_TASK_NAME, Jar::class.java)
+        val cpkPath = cpkTask.flatMap(Jar::getArchiveFile)
         val allCPKs = project.objects.fileCollection().from(cpkPath, cpbPackaging)
         val cpbTaskProvider = project.tasks.register(CPB_TASK_NAME, CpbTask::class.java) { cpbTask ->
             cpbTask.dependsOn(cpbResolution)
```

Comment:
Given that we no longer need to build the CPK from the JAR via an extra task, I think this can become
```kotlin
cpbTask.dependsOn(cpbPackaging.buildDependencies)
```
and we can remove `cpbResolution` entirely.

Reply:
Done